core/services/install-services.sh

```diff
@@ -35,6 +34,7 @@ SERVICES=(
     ping
     versionchooser
     wifi
+    ardupilot_manager # ardupilot_manager's setup.py requires the dependencies it is supposed to install...
```

Comment:
how so ? ardupilot_manager does not have direct dependency to the others services.

Reply:
setup.py tries to use our settings service, which requires pydantic

Reply:
And that'll be installed by setup.py on the following loop. My questions is, why the order of ardupilot_manager was changed since there is no dependency to others services.

Reply:
this is the reason:
https://github.com/bluerobotics/BlueOS/actions/runs/6176088878/job/16764395632?pr=2093#step:7:12948

I'm open to alternative solutions

Reply:
Check here: https://github.com/bluerobotics/BlueOS/blob/master/core/services/install-services.sh#L41

Reply:
updated